### ISSUE TITLE
Cannot navigate menu using keyboard only

### ISSUE THREAD

**Issue by vidartf** [desktop/desktop#11007]
### Describe the bug

I wanted to go to the "Repository" -> "Open in Powershell" using the keyboard only (the shortcut is not really easy to do on my keyboard layout). I tried to do it using accelerators (e.g. Alt+R opens the "Repository" menu), but the "Open in Powershell" submenu doesn't have an accelerator. I also tried to navigate the menu using Tab and/or the arrow keys, and while I was able to open the "Repository" menu, I was unable to select the menu item.

### Version & OS

Desktop version: 2.5.7
OS: Windows 10 Pro 2004.

### Steps to reproduce the behavior

See description above.

### Expected behavior

Easy menu navigation with keyboard only.

### Actual behavior

Inaccessible menus. 😞

**Comment by niik** [comment]
Hey @vidartf, thanks so much for opening this. I was able to reproduce the behavior you mention and it must be a regression because the menus on Windows are definitely supposed to be keyboard accessible.

I've marked this as a valid bug and tentatively made it a priority-2 issue.

**Comment by niik** [comment]
Hey @vidartf, I was able to narrow down what caused this and opened #11010 to fix it. I am afraid I don't know of any workarounds that doesn't involve using a pointer device at the moment but the fix will be included in the next production release which should be out next week. Sorry for the inconvenience!

**Comment by vidartf** [comment]
No worries, not a blocker for me, just figured it should be fixed so that it doesn't block in the future :)
